test/cql-pytest/test_wasm.py

```diff
@@ -1088,5 +1088,14 @@ def test_drop(cql, test_keyspace, table1, scylla_with_wasm_only, metrics):
             assert(get_metric(metrics, 'scylla_user_functions_cache_instace_count_any') > 0)
         assert(get_metric(metrics, 'scylla_user_functions_cache_instace_count_any') == 0)
 
-
-
+# Test that we can use counters as the return type of a WASM UDF.
+def test_counter(cql, test_keyspace, scylla_only):
+    schema = "p int, c counter, PRIMARY KEY (p)"
+    ri_name = unique_name()
+    wat_path = os.path.realpath(os.path.join(__file__, '../../resource/wasm/return_input.wat'))
```

Comment:
Instead of duplicating this wat_path line, it probably makes sense to assign it in the file scope once, probably right below the comment explaining how to compile that file.
Even better, since the open() and read() and replace() are also duplicated, you can create a function that returns the output of the read, and use it in the two places.

Reply:
I extracted the code to a separate function. I also used it in some other test cases, so I added it as a separate commit